manifests/init.pp

```diff
@@ -686,6 +696,20 @@
     }
   }
 
+  if $rpm_gpg_keys != undef {
+    case $rpm_gpg_keys_hiera_merge {
+      true:    { $rpm_gpg_keys_real = hiera_hash('yum::rpm_gpg_keys') }
```

Comment:
hiera functions are deprecated in favor of `lookup()`. This function can set if the lookup should be a merge like hiera_hash() is as well and validate the data so we don't have to use the validate_hash() function.

https://puppet.com/docs/puppet/5.3/hiera_automatic.html

Reply:
Changed in "Use lookup() in favor of hiera_hash()".